bench/wtperf/runners/checkpoint_stress.wtperf

```diff
@@ -0,0 +1,19 @@
+# A stress configuration to create long running checkpoints while doing a lot
+# of updates.
+conn_config="cache_size=10GB,log=(enabled=false)"
+table_config="leaf_page_max=32k,internal_page_max=16k,allocation_size=4k,split_pct=90,type=file"
+# Enough data to fill the cache. 100 million 1k records results in two ~6GB
+# tables
+icount=100000000
+create=true
+compression="snappy"
+populate_threads=1
+checkpoint_interval=60
+checkpoint_threads=1
+report_interval=5
+run_time=600
```

Comment:
Most configurations that are in the code base use a shorter runtime than this - I'm inclined to add a comment explaining that the longer runtime is important to ensure checkpoints are completing.